### PR TITLE
Add APEL Record/Message documentation for grid

### PR DESCRIPTION
**Pull request by gregcorbett**
Add APEL Record/Message documentation for grid accounting: 
- individual job accounting
- summary accounting
- sync records

Based on the content from the EGI Wiki - which can be accessed via The Internet Archive: http://web.archive.org/web/20210411151958/https://wiki.egi.eu/wiki/APEL/MessageFormat

I've added this to the internal section of the documentation, as if providers are using "off the shelf" components - they don't need to be overly concerned with the exact format of the record / message.

### CODE REVIEW

**Comment by gregcorbett** [comment]
Maybe one for the APEL team to look at as well as part of the review.

**Comment by gwarf** [comment]
Thanks @gregcorbett, please look at addressing the lint reports.

[tofu-rocketry](https://github.com/tofu-rocketry), can you please have a look?

**Comment by gregcorbett** [comment]
Lint addressed.

If I'm interpreting the `Check links` [output](https://github.com/EGI-Federation/documentation/actions/runs/10794318708/job/29938326099?pr=672), the failure isn't due to this PR. It seems to be due to the `content/en/providers/cloud-compute/openstack/aai/_index.md` page.

**Comment by sebastian-luna-valero** [comment]
Yes, thanks!

These were fixed in https://github.com/EGI-Federation/documentation/pull/673

So merging with the latest `main` should fix it

File: content/en/internal/accounting/record-and-message-formats/grid-accounting/_index.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+---
+title: "Grid Accounting"
+weight: 20
```
Comment (gwarf) [inline content/en/internal/accounting/record-and-message-formats/grid-accounting/_index.md:3]:
```suggestion
title: "Grid Accounting record and message formats"
linkTitle: "Grid"
weight: 20
```

Comment (gwarf) [reply content/en/internal/accounting/record-and-message-formats/grid-accounting/_index.md:3]: I think we can shorten the link title, and have a longer page title.

File: content/en/internal/accounting/record-and-message-formats/_index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Record and Message Formats"
+weight: 20
+type: "docs"
+description: "EGI Accounting record and message formats"
+---
+
+## Grid Accounting
+
```
[the remark this answers is not in the document]
Comment (gregcorbett) [reply content/en/internal/accounting/record-and-message-formats/_index.md:9]: done 😄

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**